ReflectometryServer/test_modules/test_pv_wrapper.py

```diff
@@ -69,6 +69,7 @@ def test_GIVEN_base_pv_WHEN_creating_motor_pv_wrapper_THEN_velocity_is_initialis
     def test_GIVEN_base_pv_WHEN_creating_motor_pv_wrapper_THEN_max_velocity_is_initialised_correctly(self):
         expected_max_velocity = self.vmax
 
+        self.wrapper.initialise()
```

Comment:
Added an initialise as previously the max velocity had been set in the init of the parent class.